cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -97,8 +99,8 @@ CUB_NAMESPACE_BEGIN
  */
 template <int NUM_ACTIVE_CHANNELS, typename CounterT, typename OffsetT>
 CUB_DETAIL_KERNEL_ATTRIBUTES void DeviceHistogramInitKernel(
-  ArrayWrapper<int, NUM_ACTIVE_CHANNELS> num_output_bins_wrapper,
-  ArrayWrapper<CounterT*, NUM_ACTIVE_CHANNELS> d_output_histograms_wrapper,
+  ::cuda::std::array<int, NUM_ACTIVE_CHANNELS> num_output_bins_wrapper,
```

Comment:
I believe this is not equivalent. We were passing a wrapper before and now we are passing an array by value. 

I believe the correct transformation would be to pass a `::cuda::std::span`

Reply:
The wrapper stores the array by value, so the array data was copied into the kernel before.

```c++
template <typename T, int COUNT>
struct ArrayWrapper
{
  /// Statically-sized array of type \p T
  T array[COUNT];
  /// Constructor
  _CCCL_HOST_DEVICE _CCCL_FORCEINLINE ArrayWrapper() {}
};
```

But maybe I am missing something.